cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -596,7 +610,7 @@ public void getArtifactTags(HttpRequest request, HttpResponder responder,
                               @PathParam("artifact-name") String artifactName,
                               @PathParam("artifact-version") String artifactVersionStr,
                               @QueryParam("scope") String scope)
-    throws BadRequestException, NotFoundException {
+    throws Exception {
```

Comment:
What else does this throw, other than the three that the one's before it are throwing?